### PR TITLE
Update ChIP-Seq tutorial with latest tool versions and links

### PR DESCRIPTION
**Pull request by pavanvidem**

### CODE REVIEW

File: topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md [hunk 1]
```diff
@@ -306,7 +307,7 @@ Since in this tutorial we are interested in assessing H3K4me3, H3K27me3 and CTCF
 >    ```
 >
 > 3. Rename the files
-> 3. **multiBamSummary** {% icon tool %} with the following parameters
+> 3. {% tool [multiBamSummary](toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_multi_bam_summary/deeptools_multi_bam_summary/3.5.1.0.0) %} with the following parameters:
```
Comment (anuprulez) [inline topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md:310]: Can we use the latest version of the tool which is: Galaxy Version 3.5.2+galaxy0 or is there a specific reason to use one prior version?

File: topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md [hunk 2]
```diff
@@ -322,7 +323,7 @@ Since in this tutorial we are interested in assessing H3K4me3, H3K27me3 and CTCF
 >
 >    Using these parameters, the tool will take bins of 1000 bp separated by 500 bp on the chromosome X. For each bin the overlapping reads in each sample will be computed and stored into a matrix.
 >
-> 4. **plotCorrelation** {% icon tool %} with the following parameters
+> 4. {% tool [plotCorrelation](toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_plot_correlation/deeptools_plot_correlation/3.5.1.0.0) %} with the following parameters:
```
Comment (anuprulez) [inline topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md:326]: Similar comment as previously made for the `multiBamSummary` tool, can we use the latest version?

File: topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md (hunk 1 above)
Comment (pavanvidem) [reply topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md:310]: because this is the most common latest version on .org and .eu

**Comment by anuprulez** [comment]
I ran this tutorial on EU, and it works fine. I verified all the updated steps except for the very last plot that compares all the 8 pairs of samples (which is time-consuming).

history: https://usegalaxy.eu/u/kumara/h/chip-seq-1

ping @pavanvidem

**Review by gallardoalba — APPROVED** [review]

**Comment by pavanvidem** [comment]
thanks @anuprulez @gallardoalba

**Pull request merged**